recipes/kgwasflow/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "kgwasflow" %}
+{% set version = "1.2.1" %}
+
+package:
+    name: "{{ name|lower }}"
+    version: "{{ version }}"
+
+source:
+  url: "https://github.com/akcorut/kGWASflow/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: 2cb0f5ab892538521df8a56716d3d0295f40ab5bd25783e831d5157f6e00928a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  entry_points:
+    - kgwasflow = workflow.kgwasflow:main
+
+requirements:
+  host:
+    - python >=3.10.10
+    - pip
+  run:
+    - python >=3.10.10
+    - mamba
+    - numpy =1.23.5
+    - pandas =1.5.3
+    - snakemake =7.25.0
+    - click
+
+test:
+  commands:
+    - kgwasflow --help
+    - kgwasflow --version
```

Comment:
Please add ` - kgwasflow test ....` as appropriate to the tests that get run, if the test dataset is small enough to run in ~1 minute or so.